src/Reflow/Parser.hs

```diff
@@ -22,12 +22,20 @@ parseFile t = either (return . parserError t) id
 parseContent :: Parser [Content]
 parseContent = do
     h <- option [] (try $ many header)
-    c <- many (quoted <|> try codeBlock <|> try pgpBlock <|> normal)
+    c <- many (blank <|> quoted <|> try codeBlock <|> try pgpBlock <|> normal)
     void eof
     return $ h <> c
 
 normal :: Parser Content
-normal = Normal <$> singleLine
+normal = do
+    ws <- many (char ' ')
+    l <- singleLine
+    return $ Normal (length ws) l
+
+blank :: Parser Content
+blank = do
+    _ <- try $ trailingWhitespace >> eol
+    return Blank
```

Comment:
Yeah, it needs some parens and a `return` as well:

```
blank = (try $ trailingWhitespace >> eol) *> return Blank
```
I kinda find that harder to read than this, so I think I'm going to leave it this way (using `void`).